app/views/dashboards/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Vesta</h1>
+<% unless current_user.student? %>
+  <% if @draws.empty? %>
+    <p>Welcome to Vesta! Use the navbar above to manage housing inventory and create draws!</p>
+  <% else %>
+    <% @draw_hash.each do |draw, hash| %>
+      <div class="card">
+        <div class="card-divider">
+          <h4><%= link_to draw.name, draw_path(draw) %> (<%= draw.status %>)</h4>
+        </div>
+        <div class="card-section">
+          <%= render hash[:partial], draw: draw, hash: hash %>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+<% end %>
+<% unless current_user.admin? %>
+  <div>
```

Comment:
This should definitely be wrapped in a card for consistency with the heading "Welcome to Vesta" (as discussed).